### ISSUE TITLE
Captive portal SSID and pass are wrong on OpenEVSE LCD when in captive portal mode

### ISSUE THREAD

**Issue by DrFrankReade** [OpenEVSE/ESP32_WiFi_V3.x#89]
On booting with a failed WiFi connection, the ESP-32 shows the wrong captive portal SSID and password on the OpenEVSE LCD.  

SSID: OpenEVSE
Pass: openevse

Seems to be hard coded in net_manager.cpp

`lcd_display(F("SSID: OpenEVSE"), 0, 0, 0, LCD_CLEAR_LINE);
  lcd_display(F("Pass: openevse"), 0, 1, 15 * 1000, LCD_CLEAR_LINE);`

**Comment by glynhudson** [comment]
Good point, we the SSID name needs to be updated on the LCD. We will get this fixed. 

I'm unable to replicate the open AP issue as mentioned in #88. Password shown on the LCD is correct for me.
